libs/execution_context/ebpf_core.c

```diff
@@ -1235,17 +1255,15 @@ _ebpf_core_get_current_cpu()
     return ebpf_get_current_cpu();
 }
 
-static void*
-_ebpf_core_map_find_and_delete_element(_Inout_ ebpf_map_t* map, _In_ const uint8_t* key)
+static int
+_ebpf_core_csum_diff(
+    _In_reads_bytes_opt_(from_size) const void* from,
+    int from_size,
+    _In_reads_bytes_opt_(to_size) const void* to,
+    int to_size,
+    int seed)
 {
-    ebpf_result_t retval;
-    uint8_t* value;
-    retval = ebpf_map_find_entry(
-        map, 0, key, sizeof(&value), (uint8_t*)&value, EBPF_MAP_FLAG_HELPER | EPBF_MAP_FIND_FLAG_DELETE);
-    if (retval != EBPF_SUCCESS)
-        return NULL;
-    else
-        return value;
+    return ebpf_csum_diff(from, from_size, to, to_size, seed);
```

Comment:
Why do we need both _ebpf_core_csum_diff and ebpf_csum_diff?   Why not just make ebpf_core_csum_diff non-static and have just one on the stack?

Reply:
I did that just for uniformity. But that's not a strong technical reason. So keeping a single function that is callable from unit test app.